changelog/3677.feature.rst

```diff
@@ -0,0 +1 @@
+Updates ``map.draw_rectangle`` API to be consistent with ``map.submap``
```

Comment:
```suggestion
Updated `sunpy.map.GenericMap.draw_rectangle` API to be consistent with `sunpy.map.GenericMap.submap`.
```

Reply:
I forgot the full import for GM.

Reply:
Ha ha, is this a GitHub bug?  The suggested difference shows up as HTML on my side.  Don't click "commit suggestion"!

Reply:
HTML?

Reply:
I see it now, I wonder how on earth that happened.